packages/lib/src/core/Analytics/utils.ts

```diff
@@ -140,18 +140,17 @@ export const getCardConfigData = (cardProps: CardConfiguration): CardConfigData
     }
 
     // Probably just for development - in real life we wouldn't expect the number of supported brands to push the endpoint limit on 128 chars
-    let brandsStr = JSON.stringify(brands);
+    let brandsStr = brands.toString();
     if (brandsStr.length > 128) {
-        brandsStr = brandsStr.substring(0, 124) + '...]';
+        brandsStr = brandsStr.substring(0, 124);
```

Comment:
I added the `+ '...]'` so it was clear from looking at the string that it had been deliberately truncated and wasn't accidentally malformed

Reply:
Hmm ok i will check with other teams, i don't think they are aware of it, and it looks difficult for the data team to program on it, because they cannot just simply use `split(',')` to split the string and get the array.

Reply:
ok... It's not vital, and is probably an edge case in development (that the brands array could be so big) - if it's causing difficulty for the data team then keep your change